armi/settings/caseSettings.py

```diff
@@ -367,10 +367,16 @@ def writeToYamlFile(self, fName, style="short"):
         style : str (optional)
             the method of output to be used when creating the file for the current
             state of settings
+        fromFile : str (optional)
+            if the source file and destination file are different (i.e. for cloning),
+            then this arg is used
         """
         self.path = pathTools.armiAbsPath(fName)
+        getSettingsPath = (
```

Comment:
I would move this inside of the `if style == "medium" that takes place right after, since this is only used in there.